BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -618,6 +629,15 @@ public void InitializationIsSkippedForSampledItem(string appDomainName = "Incorr
         [Event(66, Message = "Call to WindowsIdentity.Current failed with the exception: {0}.", Level = EventLevel.Warning)]
         public void LogWindowsIdentityAccessSecurityException(string error, string appDomainName = "Incorrect") => this.WriteEvent(66, error ?? string.Empty, this.nameProvider.Name);
 
+        [Event(67, Message = "Backend has responded with {0} status code in {1}ms.", Level = EventLevel.Warning)]
```

Comment:
This doesnt't qualify for Warning level. Its just Informtional.

Reply:
Intentionally kept it has warning, so that our Diagnostic module captures this information. Do you think we should change this to information?

Reply:
Diagnostic module captures only Error by default.
Use log level, as appropriate for the event. 
Eg: 
Backend responded - is informational.
Backend failed to respond is Warning.
SDK is dropping items is Error, and so on.